### PR TITLE
Pw 16192 fix

### PR DESCRIPTION
**Pull request by youngfreezyVA**
issue #16192
css polishes for the back to top button:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/16192

### CODE REVIEW

File: src/site/components/up_to_top_button.html [hunk 1]
```diff
@@ -12,16 +12,13 @@
 
 {% endcomment %}
 
-<div data-template="components/up_to_top_button" id="top-button-container" class="vsa-top-button-container">
+<div class="vsa-top-button-container" data-template="components/up_to_top_button" id="top-button-container">
     <div class="usa-grid usa-grid-full">
-        <div id="usa-width-container" class="usa-width-three-fourths">
```
Comment (kelsonic) [inline src/site/components/up_to_top_button.html:17]: Do you know why this was an `id` to begin with? CC @zacharymorel Just want to make sure it's not being targeted by JS

Comment (kelsonic) [reply src/site/components/up_to_top_button.html:17]: Ah, nvm I see now

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉 Haven't tried it out locally to see if it looks ✅ but I'm assuming you have!

**Comment by youngfreezyVA** [comment]
> LGTM! 🎉 Haven't tried it out locally to see if it looks ✅ but I'm assuming you have!

yup I have @kelsonic! and before I merge it in, here it is in action:
https://drive.google.com/file/d/1eRdyfGsUbtEcQwjMdzcZwrkuXaMSp2rG/view?usp=sharing
cc: @rtwell

**Comment by rtwell** [comment]
lookin' sharp—i sure wish i could see this on my own browser/device 😢 

btw make sure you ping @jenstrickland 

thanks for the hard work!

**Review by theodur — APPROVED** [review]

**Pull request merged**